### DISCUSSION TITLE
Suggestions for a New App

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#292] [General]
Hi everyone, today I've released Alpaca 2.0.0, as always, thanks for the support and contributions, I'm really happy with the success the app has had on Flathub, there's still a lot of stuff I intend to add in the future, but I also want to start thinking about a new project.

Now, I've been using Linux on and off for around 6 years and I fully switched to it 4 years ago. It's been really inspiring seeing stuff like Wayland and Steam take off making Linux a much better and complete operating system, I've always wanted to help this growth but I'm broke so the next best way of supporting Linux is by making stuff for it.

Alpaca was the first project that I actually published and developed to a stable version, but I don't intend it to be the last, I have some ideas of what I could do next, but I wanted to ask you all for some suggestions.

What should I make next?

**Comment by linux-universe** [comment]
a kind of alpaca but for stable diffusion
Alpaca has already managed to install local ai extremely easily via flatpak which is unique
Something similar with stable diffusion would be great

**Reply by Jeffser** [reply]
thanks for the idea, unfortunately I'm against image generation

**Reply by loulou64490** [reply]
use this app [https://imaginer.codeberg.page/](https://imaginer.codeberg.page/)

**Comment by olumolu** [comment]
If you want a project that is different from what you have done up to the point and I assume that you are asking for a new project in my opinion I have to projects in my mind one of them
Music application written gtk using Spotify or apple music or youtube music maybe jiomusic apis
Another gtk4 app for controlling smart home stuff light blub and other stuff

**Reply by Jeffser** [reply]
well there are a lot of music players so I don't see what I could do differently to justify making another one, and about the home assistant stuff I think there's one called butler that's written with GTK4

**Reply by olumolu** [reply]
Ok I don't know that

**Comment by loulou64490** [comment]
A good image viewer / editor in GTK 4 with OCR, a lot of cropping functionalities, compressor, etc... like [gThumb](https://gitlab.gnome.org/GNOME/gthumb) but in GTK4 and a easier interface (or like the windows 11 viewer)

**Reply by olumolu** [reply]
This is something that can be considered.
Using ocr or other ai stuff we can easily have a gimp alternative maybe not that advance but having ai into the app we can have most of the task automated. And basic stuff like crop and changeing size format and other stuff

**Comment by loulou64490** [comment]
A Google Keep client

**Reply by Jeffser** [reply]
This could be interesting, I wonder if Gnome Accounts has access to that

**Reply by loulou64490** [reply]
no, that's why there isn't app lol

**Reply by Jeffser** [reply]
Makes sense lmao

**Reply by olumolu** [reply]
If it is still i recommend not to because there is standard notes notes that encrypt files unlike keep which does nothing

**Reply by loulou64490** [reply]
personnaly idc about note encryption

**Reply by olumolu** [reply]
You should note encryption is very important mainly what we write is important so that should not be used for ads

**Reply by loulou64490** [reply]
I never see ad in my life, I always use uBlock Origin

**Reply by olumolu** [reply]
> I never see ad in my life, I always use uBlock Origin

Yes we all use but why notes encryption is needed
https://www.privacyguides.org/en/notebooks/

**Reply by loulou64490** [reply]
yeah yeah I know there is alternative, but I use Google every day (I have Google One) and I like to have all app in one service

**Comment by boukendesho** [comment]
A python module make screenshot or record screen (whole or area) or acess system clip borad simple through xdg-desktop-portal under wayland.

I see many python projects that can't support wayland because of this, current there is no such module in python.

**Reply by Jeffser** [reply]
I believe GLib is able to do this but I'm not quite sure. It's intended mostly for GTK apps tho

**Comment by Jeffser** [comment]
I might have started a little something, I wanna call it Garden, if someone guesses what it is I will show some screenies of it running

**Reply by olumolu** [reply]
A todo list app

**Reply by Jeffser** [reply]
nopeee

**Reply by loulou64490** [reply]
![Capture d'écran 2024-09-14 093426](https://github.com/user-attachments/assets/4f278bf0-5776-4308-be20-cd5fa0e0f30f)
😎

**Reply by Jeffser** [reply]
Hahaha nah, what I'm doing is kinda interactive like a game but it isn't a game

**Reply by loulou64490** [reply]
An app like [Forest](https://www.forestapp.cc/) to be focus?

**Reply by Jeffser** [reply]
Nope it does something outside the window

**Reply by loulou64490** [reply]
> Does the app already exist? 

Not really

> Is it really useful ?

Nah

> Does the app edit files?

Nope

**Reply by olumolu** [reply]
I was thinking making a gtk4 app powered by clamAV might be good a good antivirus which maybe easy to scan and give other features that like we get in windows systems why not make this.?

**Reply by loulou64490** [reply]
bro there isn't virus on linux

**Reply by linux-universe** [reply]
there are a few, but the best things against it aren't anti viruses. imo it's flatpak and other container's like distrobox

**Reply by Jeffser** [reply]
the best way of preventing viruses on Linux is to just have common sense

**Reply by olumolu** [reply]
If someone does not have 

> common sense

having this app will be a good deal

**Reply by Jeffser** [reply]
https://github.com/Jeffser/Alpaca/discussions/308 🤔

**Comment by olumolu** [comment]
Pinball like game in gtk4 with python
